core/server/data/migrations/versions/3.18/01-add-email-preview-permissions-to-roles.js

```diff
@@ -1,36 +1,39 @@
-const utils = require('../../../schema/fixtures/utils');
-const logging = require('../../../../../shared/logging');
+const {
+    combineTransactionalMigrations,
+    addPermissionToRole
+} = require('../../utils');
 
-const resource = 'email_preview';
-
-module.exports = {
-    config: {
-        transaction: true
-    },
-
-    async up(options) {
-        const relationFixtures = utils.findPermissionRelationsForObject(resource);
```

Comment:
my understanding is `findPermissionRelationsForObject` is not used any longer, so that can be removed completely from the codebase ;)

Reply:
Looks like it's still used in the tests! https://github.com/TryGhost/Ghost/blob/master/test/utils/index.js#L388